openstack/examples/os-omero.yml

```diff
@@ -31,6 +31,7 @@
     become: yes
 
   roles:
+  - role: versioncontrol-utils
   - role: upgrade-distpackages
```

Comment:
Very pedantic aesthetic point... I think `upgrade-distpackages` should be first:
1. Create an new VM
2. Upgrade existing packages
3. Install new stuff